docs/docs/reference/metaprogramming/inline.md

```diff
@@ -360,18 +362,19 @@ Then:
   val dAny: None.type = defaultValue[Any]
 ```
 
-As another example, consider the type-level version of `toNat` above the we call
-`toIntT`: given a _type_ representing a Peano number, return the integer _value_
-corresponding to it. Consider the definitions of numbers as in the _Inline
-Match_ section aboce. Here's how `toIntT` can be defined:
+As another example, consider the type-level version of `toInt` below: 
+given a _type_ representing a [Peano number](https://wiki.haskell.org/Peano_numbers),
```

Comment:
```suggestion
given a _type_ representing a Peano number,
```

The Haskel version works on term rather that terms and types. This will be miseading to someone trying to understand the encoding.